README.md

```diff
@@ -1,8 +1,62 @@
+
+## Skills
+
+**Proficient:** Java, Python, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** Docker, HTML/CSS, Maven, Selenium, BigQuery(Google Cloud Platform), Dataproc(Google Cloud Platform)
+
+**Familiar:** EC2 (Amazon Web Services), Pandas, Numpy, DynamoDB, Hadoop
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_PayalGupta](https://github.com/jarviscanada/jarvis_data_PayalGupta)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_PayalGupta/tree/master/linux_sql)]: Implemented a Linux clustering monitoring agent to monitor, collect and analyze data from host machines in a 10 node cluster over a network in a PostgreSQL database provisioned using docker container using bash scripts for better business decisions and resources planning.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_PayalGupta/tree/master/core_java)]:
+      
+  - Twitter App: Developed a CLI interface using MVC architecture for users to create, read and delete tweets through Twitter RESTful API. Performed testing using JUnit test cases.
+  - JDBC App: Constructed the DAO implementation of retail customer orders through JDBC with PostgreSQL. The data manipulation through CRUD was tested.
+  - Grep App: Implemented Grep application in Java which simulates the Linux grep command to find patterns in a file and write them into another file. The implemented code was then optimized using Lambda and Streams API
+
+**Springboot App** [[GitHub](https://github.com/jarviscanada/jarvis_data_PayalGupta/tree/master/springboot)]: Not Started
+
+**Python Data Analytics** [[GitHub](https://github.com/jarviscanada/jarvis_data_PayalGupta/tree/master/python_data_anlytics)]: Not Started
+
+**Hadoop** [[GitHub](https://github.com/jarviscanada/jarvis_data_PayalGupta/tree/master/hadoop)]: Not Started
+
+**Spark** [[GitHub](https://github.com/jarviscanada/jarvis_data_PayalGupta/tree/master/spark)]: Not Started
+
+**Cloud/DevOps** [[GitHub](https://github.com/jarviscanada/jarvis_data_PayalGupta/tree/master/cloud_devops)]: Not Started
+
+
+## Highlighted Projects
+**HR Management System**: Developed an HR Management System in Java using JDBC, MySQL, Servlets, JavaScript. The application supported both user and HR login, the user had the access to their employment details as well as the privilege to edit their information and the HR had access to view all users with privileges to add, remove and edit a user.
+
+
+## Professional Experiences
+
+**Software Developer, Jarvis (2020-present)**: Implemented various projects successfully using agile methodologies involving SQL, Bash, Java, Maven, Docker in Linux(CentOS and Ubuntu) and WSL. Participated in scrum ceremonies and collaborated with team members.
```

Comment:
talk about the following points 
Worked in an agile/scrum team of x people.
responsible for developing xyz using abc, bcd, etc..
talk about 1-2 soft skills  (e.g. leadership, collaboration, communication, etc.)

Reply:
overall, the description is too short.